client/src/css/stylesheet.scss

```diff
@@ -35,177 +42,315 @@ body {
       display: flex;
       align-items: center;
 
+      #status-bar {
+        width: auto;
+        height: 100%;
+        text-align: center;
+        flex-grow: 1;
+
+        #status {
+          position: relative;
+          top: 50%;
+          transform: translateY(-50%);
+          margin: 0;
+          font-size: 2em;
+        }
+      }
+    }
+
+    #left-flex {
+      position: absolute;
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: center;
+      align-content: space-between;
+      align-items: space-between;
+      left: 0%;
+      top: 12%;
+      width: 50%;
+      height: 88%;
+      overflow: visible;
+
       #left-arrow,
       #right-arrow {
-        width: 7%;
+        width: 5%;
       }
 
       #left-arrow {
-        height: 100%;
-        text-align: center;
+        position: relative;
```

Comment:
``body #dash-wrapper #left-flex #left-arrow`` and ``body #dash-wrapper #left-flex #right-arrow`` share these 3 declarations

Would it make sense to structure this so that we apply a rule to both arrows?

  - height: 20%
  - position: relative
  - top: 0%